common/primitives/core/src/teebag/types.rs

```diff
@@ -70,8 +70,9 @@ pub enum AttestationType {
 #[derive(Encode, Decode, Clone, Copy, Default, PartialEq, Eq, RuntimeDebug, TypeInfo)]
 pub enum WorkerType {
     #[default]
+    #[codec(index = 0)]
     Identity,
-    BitAcross,
+    #[codec(index = 2)]
```

Comment:
Shall we use `index = 1` to avoid any surprises? Or will it affect any existing storages? At least not on prod I assume